RocketChat/Rocket.Chat.ReactNative
# [IMPROVE] Create edited component for edited messages

## Proposed changes
Today when a message is edited after it was sent, we show an `edited` text at the end of it. 
The proposal of this PR is change this behavor to show an `icon pen` as web do it. 

We also change the current icons showed at left when the message doesn't have a header. E.g: encrypeted or error icons. To show on left, centralized at the top, (take a look at the screenshots section).

Related PR: https://github.com/RocketChat/commonmark-react-renderer/pull/1


## Issue(s)	


## How to test or reproduce
* edit a message; 
* simulate an error to show the `error icon`;
* encrypted messages to show `encrypted icon`;
* enable the `read message` option, to show the `blue check icon` when the message was read;

OBS: test on tablets and IPads too,

## Screenshots
| Before | After |
|--------|------|
| ![Simulator Screen Shot - iPhone 12 - 2022-04-13 at 15 27 25](https://user-images.githubusercontent.com/14894380/163246284-131a95a2-1503-41cb-b4a3-8b9a01e81fc7.png) | ![Simulator Screen Shot - iPhone 12 - 2022-04-13 at 15 27 14](https://user-images.githubusercontent.com/14894380/163246332-ed0599fa-0205-4357-b63e-9b4012e480dc.png) |
| ![Simulator Screen Shot - iPhone 12 - 2022-04-12 at 12 28 36](https://user-images.githubusercontent.com/14894380/163246563-e4d987f8-460e-4fd3-91a6-13925a5c0ca8.png) | ![Simulator Screen Shot - iPhone 12 - 2022-04-12 at 13 16 27](https://user-images.githubusercontent.com/14894380/163246813-7dd6e0f7-c48b-4d1c-853b-65177ce438e5.png) |


## Types of changes




## Checklist



## Further comments
